src/blog/2024/03/http-authentication-node-red-with-flowfuse.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Securing HTTP Traffic for Node-RED with FlowFuse
+subtitle: Choosing the right form of authentication for your Node-RED integration is important.
+description: In this blog we will discuss the various forms of authentication for integrating web applications with Node-RED.
+date: 2024-03-25
+authors: ["grey-dziuba"]
+image: 
+tags:
+    - posts
+    - flowfuse
+    - http in
+    - http node
+    - api
+---
+
+Citizen development empowers employees to create digital solutions. However, it requires guardrails to ensure data security, operational stability, and compliance. These guardrails are what FlowFuse provides to the Node-RED community to level up their deployments. FlowFuse offers many different security measures for authentication and authorization, which all apply to different scenarios. 
```

Comment:
My understanding of English is that it would be guard rails, versus guardrails. I'm not sure, but letting you make a decision. Just flagging